responses/fr/HassVacuumStart.yaml

```diff
@@ -0,0 +1,5 @@
+language: fr
+responses:
+  intents:
+    HassVacuumStart:
+      default: "Démarré"
```

Comment:
I am ok with `Démarré` but we have an opportunity for something a be more .... "cute" and "smart"
Because we know that it's a vacuum that is starting, we could say
`Nettoyage lancé`
I am fine with both